GA-worksheet.md

```diff
@@ -72,24 +89,28 @@
     }
 }
     ```
+    
+This media query will not be activated unless the browser is smaller than 500px.
 
 6. What does each digit of this Hexidecimal Color Code represent? #E78
 
- E represents ...
+ E represents a high level of red
 
- 7 represents ...
+ 7 represents a level of greeness
 
- 8 represents ...
+ 8 represents a level of blueness
 
-7. What is the Hex Color Code for true purple?  #
+7. What is the Hex Color Code for true purple?  #609
 
-8. _______________ is the programming language that allows websites to be interactive.
+8. Javascript is the programming language that allows websites to be interactive.
 
-9. This tag allows us to add JavaScript into any webpage? <          >
+9. This tag allows us to add JavaScript into any webpage? < script >
 
 10. What is an event?
+An event is one result of using your mouse.
```

Comment:
Events are anything that happen to your webpage, like page load, so they are not limited to mouse events.